assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -63,6 +64,7 @@ const AlertsList: ComponentType<AlertsListProps> = ({
   } = useAlertsPageContext();
   const dispatch = useAlertsPageDispatchContext();
   const navigate = useNavigate();
+  const prevAlertIds = usePrevious(alerts)?.map((alert) => alert.id);
```

Comment:
I believe you can ensure that `prevAlertIds` is always an array by using the nullish-coalesce operator `??`:
```suggestion
  const prevAlertIds = usePrevious(alerts)?.map((alert) => alert.id) ?? [];
```